tests/e2e/scale/noobaa/test_hsbench.py

```diff
@@ -3,7 +3,11 @@
 from ocs_ci.ocs import hsbench
 from ocs_ci.utility import utils
 from ocs_ci.framework.testlib import E2ETest, scale
-from ocs_ci.framework.pytest_customization.marks import vsphere_platform_required
+from ocs_ci.framework.pytest_customization.marks import (
+    vsphere_platform_required,
```

Comment:
This test also can be run on both aws and VMware clusters.

Reply:
True, I only added skip for existing RGW test for platforms other than vsphere. Skip was at class level.